src/Test/Utilities/Shared/Metadata/ILBuilderVisualizer.cs

```diff
@@ -70,7 +69,7 @@ private static List<HandlerSpan> GetHandlerSpans(ImmutableArray<Cci.ExceptionHan
             {
                 HandlerSpan span;
 
-                if (region.HandlerKind == ExceptionRegionKind.Filter)
+                if (region.HandlerKind == System.Reflection.Metadata.ExceptionRegionKind.Filter)
```

Comment:
You can add 
```using System.Reflection.Metadata;``` inside namespace definition in the file to resolve that ambiguity. That's the general pattern used elsewhere until we move to SRM v1.3